source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -23,31 +23,50 @@ This is the normal order we use when upgrading clusters, it does not have to exa
 - Production       - (prod-00-aks, prod-01-aks/ ss-prod-00-aks, ss-prod-01-aks) 
 - Ptl              - (cft-ptl-00-aks/ ss-ptl-00-aks) 
 
-## Pods and HRs Health Checks
+## Planning
 
-Please post a notification in the #cloud-native-announce channel to notify people which clusters will be upgraded (Please refrain from using @here, as this is intended for notifications rather than announcements).
-Utilize the #sds-cloud-native channel if you are involved in activities concerning SDS clusters. 
+Before starting to upgrade any clusters, we need to find out if the version of kubernetes we are upgrading to has any deprecations in it that might affect our applications.
+
+We can do this using the  [pluto tool](https://github.com/FairwindsOps/pluto).
 
-Ensure your logged into the correct cluster.
+Ensure you're logged into the correct cluster.
 
 ```command
 kubectl config current-context
 ```
 
-Before upgrading any of the clusters run the following query on the cluster you are planning to upgrade to find out the statuses of the pods and hrs:
+Run pluto against the prod and ptl clusters to ensure there are no resources using apiVersions that will be deprecated in the kubernetes version you are upgrading to.
```

Comment:
Small thing again sorry, just prod and ptl?

Is it worth adding here that pluto should be run against all envs so we aren't blocked in any of them? Some clusters might have resources that others don't

Reply:
The thinking was that we should only check PTL and Prod since if it's there, it's likely to be in the others as well. I did add a bit saying if deprecations are found in those two environments, to check all of them after so we can determine what apps are using deprecated versions. @thomast1906 what you do think?

Reply:
Could do yeah, it's not much extra work to check tbf

Reply:
The counter argument is that if it works in prod, we should go ahead since we only really care about prod in a sense. Although, I'd like to think if there is a deprecation in one environment an application is deployed to, it should exist in all since we keep them all in sync version wise.

Reply:
Also true 😁 Though we don't usually do prod til the end which could block us if some other env was blocked before then

I think in the past we had this exact case where something was deployed only in demo / pinned in demo and we had to change it in flux, but it would've blocked the AKS upgrade